### DISCUSSION TITLE
Problem on installing in M1 bigsur

### DISCUSSION THREAD

**Discussion by Alessandro2301** [Homebrew/discussions#438] [Getting started] [answered]
Hi, i installed homebrew how the site explain but when the installation finished, i had tried to run 'brew help' or 'brew install wget' or anyone other brew command but the terminal always answered 'zsh: command not found: brew'.
What can i do now? It's the first homebrew's installation for me so i'm not so expert.

Thanks!

**Comment by carlocab** [comment, marked as answer]
You need to add Homebrew to your `PATH`. Do
```
echo 'export PATH="/opt/homebrew/bin:$PATH"' >> .zshrc
```
and then restart your shell. (You'll need to replace `.zshrc` with the appropriate `rc` file if you're not using Zsh. Zsh is the default shell on macOS.)

**Reply by Alessandro2301** [reply]
Perfect, thanks!

**Reply by jorge23gg** [reply]
Di it work for you? I copied and pasted but error persists, also when checking what is inside the opt/ folder I don't see Homebrew. However it says installation was successful

**Reply by jorge23gg** [reply]
oh sorry I didn't restart properly the shell, thanks!
